integration-cli/docker_utils.go

```diff
@@ -1567,3 +1567,23 @@ func sumAsIntegers(vals ...interface{}) interface{} {
 	}
 	return s
 }
+
+// pollFunc is used to periodically execute a check function, it
+// returns error after timeout.
+func pollFunc(f func() error, timeout time.Duration) error {
+	if f() == nil {
```

Comment:
I think it's checking for the error returned by `f()` :stuck_out_tongue_closed_eyes: